src/server/game/Entities/Player/Player.h

```diff
@@ -2906,6 +2916,7 @@ class Player : public Unit, public GridObject<Player>
     time_t m_resetTalentsTime;
     uint32 m_usedTalentCount;
     uint32 m_questRewardTalentCount;
+    uint32 m_otherBonusTalentCount;
```

Comment:
@zxdhe121 don't forget to rename this too for consistency :)

```c++
uint32 m_extraBonusTalentCount
```

Reply:
@FrancescoBorzi Embarrassed... fixed.Thx again.